jenkins-pipelines/manager/centos-manager-backup-azure.jenkinsfile

```diff
@@ -5,11 +5,14 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 managerPipeline(
     backup_bucket_backend: 'azure',
-    backend: 'aws',
+    backend: 'azure',
     ip_ssh_connections: 'public',
-    region: '''["us-east-1", "us-west-2"]''',
+    azure_region_name: 'eastus',  //TODO: modify to multy region once sct supports more than one region
     test_name: 'mgmt_cli_test.MgmtCliTest.test_backup_feature',
-    test_config: 'test-cases/manager/manager-regression-multiDC-set-distro.yaml',
+    test_config: 'test-cases/manager/manager-regression-azure.yaml',
+    scylla_version: '',
+    azure_image_db: '/subscriptions/6c268694-47ab-43ab-b306-3c5514bc4112/resourceGroups/scylla-images/providers/Microsoft.Compute/images/ScyllaDB-5.1.dev-0.20220717.4bfcead2ba60-1-build-201',
+    // scylla master, since scylla 5.0 doesn't support azure yet, and in fact failed to start at all
```

Comment:
`5.0.0` doesn't support it, but `5.0.1` does support it, and IIUC it should be released today (or perhaps was released yesterday

Reply:
switched to 5.0.2